File: tests/expected/coverage.jl

function indexing()::Int64
    sum = 0
    a::Array{Int64} = []
    for i in (0:10-1)
        push!(a, i)
        sum += a[i+1]
    end
    return sum
end

function show()
    a1 = 10
    a2::Float64 = 2.1
    println(join([a2], " "))
    for i in (0:10-1)
        println(join([i], " "))
    end
    for i in (0:2:10-1)
        println(join([i], " "))
    end
    a3 = -(a1)
    a4 = (a3 + a1)
    println(join([a4], " "))
    sum1 = indexing()
    println(join([sum1], " "))
end

function main()
    show()
end

main()

File: tests/expected/coverage.rs
use std::collections;

fn indexing() -> i32 {
    let mut sum: i32 = 0;
    let mut a: Vec<i32> = vec![];
    for i in (0..10) {
        a.push(i);
        sum += a[i as usize];
    }
    return sum;
}

fn show() {
    let a1: i32 = 10;
    let a2: f32 = 2.1;
    println!("{}", a2);
    for i in (0..10) {
        println!("{}", i);
    }
    for i in (0..10).step_by(2) {
        println!("{}", i);
    }
    let a3: i32 = -(a1);
    let a4: i32 = (a3 + a1);
    println!("{}", a4);
    let sum1: i32 = indexing();
    println!("{}", sum1);
}

fn main() {
    show();
}
